mautic/mautic
# Add repeatable points action

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Points action are executable just one time. 
This PR allow repeat executions for points action, for example visits a page. 

![image](https://user-images.githubusercontent.com/462477/43385229-73075afe-93e0-11e8-88bc-f52c59112ff4.png)



#### Steps to test this PR:
1.  Create Visits specific URL Point action
2.  Execute action more times
3.  See action was execute just one time
4.  Enable new option: Allow repeat the action
5. Execute action one or more times
6. See IF on each run of action, points are executable
